test/integration/targets/docker_network/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create random name prefix
+  set_fact:
+    name_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+    cnames: []
+    dnetworks: []
+
+- debug:
+    msg: "Using name prefix {{ name_prefix }}"
+
+- block:
+  - include_tasks: run-test.yml
+    with_fileglob:
+    - "tests/*.yml"
+
+  always:
+  - name: "Make sure all containers are removed"
+    docker_container:
+      name: "{{ item }}"
+      state: absent
+      stop_timeout: 1
+    with_items: "{{ cnames }}"
```

Comment:
You might want to use `loop` here (it is already used in `basic.yml`).

Reply:
Thanks! Fixed.